javascript-arrays.tar/javascript-arrays/main.js

```diff
@@ -0,0 +1,12 @@
+const colors = ['red', 'white', 'blue'];
+console.log(colors[0]);
+console.log(colors[1]);
+console.log(colors[2]);
+console.log(
+  'America is ' + colors[0] + ', ' + colors[1] + ', ' + 'and ' + colors[2]
+);
+colors.splice(2, 0, 'green');
```

Comment:
another simple way to do this is to just overwrite the existing value at index 2 with the assignment operator like this: `colors[2] = 'green';`